e2e/chatqna/README.md

```diff
@@ -0,0 +1,27 @@
+# OPEA ChatQnA workload on OCP
+
+## Overview
+This workload is based on the [OPEA ChatQnA](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA/deprecated) with [Intel® Gaudi 2 accelerator](https://habana.ai/products/gaudi2/) and [Red Hat OpenShift AI](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai) (RHOAI) on RHOCP. It uses RAG with Redis vector-db and text generation inference server on RHOAI and Intel Gaudi2 accelerator. Refer to the OPEA readme for more details about the [workload architecture](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA#chatqna-application). 
+
+## Prerequisites
+* Provisioned RHOCP cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster)
+* Provisioning Intel Gaudi 2 accelerator:
+  * Setup KMM and node feature discovery (NFD) for drivers. Follow the steps [here](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/Environment_Setup.html)
+  * Setup HabanaAI operator and create device config custom Resource. Follow the steps [here](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/Deploying_HabanaAI_Operator.html)
+* Install RHOAI. Follow steps [here](../inference/README.md/#install-rhoai)
+
+## Deploy workload
+* The workload uses docker-compose yaml's. They can be converted into OCP compatible yaml's using [Kompose tool](https://kubernetes.io/docs/tasks/configure-pod-container/translate-compose-kubernetes/). This creates the corresponding deployment and service yaml's.
```

Comment:
Do we really need this?

Reply:
docker-compose files are not directly compatible on k8s/ocp. this tool converts to k8s worklods (deploymentconfig and service)